config/initializers/notifications.rb

```diff
@@ -0,0 +1,7 @@
+ActiveSupport::Notifications.subscribe('external_error') do |_name, _start, _finish, _id, payload|
```

Comment:
NewRelic is only on the nucore-nu branch. Do you want me to wait until this is merged to the nucore-nu branch before I add that?